contracts/optimistic-ethereum/iOVM/chain/iOVM_CanonicalTransactionChain.sol

```diff
@@ -60,6 +63,19 @@ interface iOVM_CanonicalTransactionChain {
      * Public Functions *
      ********************/
 
+    /**
+     * Accesses the queue storage container.
+     * @return Reference to the queue storage container.
+     */
+     // @todo: uncommenting this line breaks `yarn build`, but not `solc`. 
+     // must be something in the ovm-compiler, but why?
+    // function queue()
+    //     external
+    //     view
+    //     returns (
+    //         iOVM_ChainStorageContainer
+    //     );
```

Comment:
My best guess: the `ovm-compiler` plugin doesn't handle errors as gracefully, but under the hood, it is complaining that `queue()` is not marked `override` in the implementation.

Reply:
Correct, that resolved it.